app/assets/javascripts/discourse/app/models/draft.js

```diff
@@ -25,16 +25,20 @@ Draft.reopenClass({
 
   save(key, sequence, data, clientId, { forceSave = false } = {}) {
     data = typeof data === "string" ? data : JSON.stringify(data);
-    return ajax("/draft.json", {
-      type: "POST",
-      data: {
-        draft_key: key,
-        sequence,
-        data,
-        owner: clientId,
-        force_save: forceSave,
+    return ajax(
+      "/draft.json",
+      {
+        type: "POST",
+        data: {
+          draft_key: key,
+          sequence,
+          data,
+          owner: clientId,
+          force_save: forceSave,
+        },
       },
-    });
+      false
```

Comment:
I'm not entirely familiar with our ajax library but could we pass `ignoreUnsent` as another item in the second argument object? I think it would be easier to read.

Reply:
I was thinking about it too, but the arg object is the object that we eventually [pass](https://github.com/discourse/discourse/blob/ec8edf8f7a02b2ce3adef9e24d34be55dcbb618f/app/assets/javascripts/discourse/app/lib/ajax.js#L170) into jQuery [ajax](https://api.jquery.com/jquery.ajax/) function.

I'd like us to avoid mixing settings specific for our app into this object. It'll be harder to read and support.

Reply:
Yes, fair enough. We could delete it before passing it to jQuery's `ajax()`, maybe? If you think it's useful, I don't insist on it.

Reply:
>We could delete it before passing it to jQuery's ajax(), maybe?

Hmm, I see now that we already do this with other fields. I guess it's ok, it's more important to have a clean code when we call our `ajax` function`. I've changed it.